Makefile

```diff
@@ -30,3 +30,23 @@ bin/goose: $(shell find . -type f -name '*.go')
 .PHONY: clean
 clean:
 	rm -rf bin
+
+# Check that given variables are set and all have non-empty values,
+# die with an error otherwise.
+#
+# Params:
+#   1. Variable name(s) to test.
+#   2. (optional) Error message to print.
+#
+# cf: https://stackoverflow.com/questions/10858261/abort-makefile-if-variable-not-set
+check_defined = \
+	$(strip $(foreach 1,$1, \
+		$(call __check_defined,$1,$(strip $(value 2)))))
+__check_defined = \
+	$(if $(value $1),, \
+		$(error Undefined $1$(if $2, ($2))))
+
+.PHONY: release
+release:
+	@:$(call check_defined, GITHUB_TOKEN)
+	docker run -e GITHUB_TOKEN=$(GITHUB_TOKEN) --rm  -v $(PWD):/workdir -w /workdir cbroglie/goreleaser-cgo:1.12.12 goreleaser --rm-dist
```

Comment:
This uses the image I built here: https://hub.docker.com/r/cbroglie/goreleaser-cgo.

It simply adds the goreleaser binary to the `dockercore/golang-cross` image which is used for cross builds of https://github.com/docker/cli. Will move it to a more official location soon.

Reply:
Doesn't have to block this but we should move this to an org owned image

Reply:
Yep, will do